docs-gen/Makefile

```diff
@@ -14,4 +8,7 @@ clean:
 	find ../docs -mindepth 1 ! -name '.nojekyll' -delete
 
 serve:
-	hugo server -b http://localhost:1313/
+	hugo server -b http://localhost:1313/kfp-operator/
```

Comment:
this is preferable as it matches the sub path we have when deploying with github pages (and it's [what the README already says we should do](https://github.com/sky-uk/kfp-operator/blob/925290ccc5c63a158732d24b68f217ccd15edb84/docs-gen/README.md?plain=1#L13) anyway!)